ale/drivers/kaguya_driver.py

```diff
@@ -30,21 +42,28 @@ def metakernel(self):
                     self._metakernel = mk
         return self._metakernel
 
+    @property
+    def ending_ephemeris_time(self):
+        if not hasattr(self, '_ending_ephemeris_time'):
+            self._ending_ephemeris_time = self.label.get('CORRECTED_SC_CLOCK_STOP_COUNT').value
+            self._ending_ephemeris_time = spice.sct2e(self.spacecraft_id, self._ending_ephemeris_time)
+        return self._ending_ephemeris_time
+
+
     @property
     def starting_ephemeris_time(self):
         if not hasattr(self, '_starting_ephemeris_time'):
             self._starting_ephemeris_time = self.label.get('CORRECTED_SC_CLOCK_START_COUNT').value
             self._starting_ephemeris_time = spice.sct2e(self.spacecraft_id, self._starting_ephemeris_time)
         return self._starting_ephemeris_time
 
-
     @property
     def _detector_center_line(self):
-        return 1
+        return self.image_lines/2
```

Comment:
I thought this was not always true, for example, sometimes the center is shifted one pixel to the left or right. Why not use similar method to _detector_center_sample?

Reply:
Yes, and after double checking, original is correct. This should be reverted.